opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -251,9 +251,11 @@ message Profile {
   int32 default_sample_type_index = 9;
 
   // A globally unique identifier for a profile. The ID is a 16-byte array. An ID with
-  // all zeroes is considered invalid.
-  //
-  // This field is required.
+  // all zeroes is considered invalid. It may be used for deduplication and signal
```

Comment:
if the ID is a 16-byte array, and is unset, then all bytes are set to zero - is a profile ID unset therefore invalid per this comment?

Reply:
I think the idea behind stating that `An ID with all zeroes is considered invalid` is just to provide a null value that the community can use in implementations when a well-formed value must be set for practical reasons.